README.md

```diff
@@ -113,168 +73,12 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
```

Comment:
* I get an error when I try to add data to your database below 👇 is a screenshot of the error, Kindly stick to this data format for your data `dd-mm-yyyy`  this would solve the error 👍 
![image](https://user-images.githubusercontent.com/52497246/214201209-6c30300a-4e4a-42ee-afc5-d716af83ab52.png)